src/data.json

```diff
@@ -1,16 +1,14 @@
 {
   "coin": [
```

Comment:
I would make a key value  here, with the key the asset hash and the value what you have now (ie. KnownAsset interface)

Reply:
fixed this